pkgs/development/libraries/gtk/4.x.nix

```diff
@@ -86,27 +83,25 @@ stdenv.mkDerivation rec {
     python3
     sassc
     gi-docgen
+    libxml2 # for xmllint
```

Comment:
Should we make an extra output for xmllint?

Reply:
Unfortunately, that currently is not really possible to do conveniently – build systems generally support only single `bindir` per project. (Eventually, I want to have Debian-level granularity packages but that would require massive redesign of Meson and porting the affected projects to Meson.) Though, I wonder if we should explicitly list the outputs used – it used to be that way in the past.